include/ccpp_adapt.h

```diff
@@ -0,0 +1,63 @@
+
+#pragma once
+
+#include "workaround.h"
+
+#if WORKAROUND_COMPUTECPP
+
+#include <CL/sycl.hpp>
+
+/*
+This whole file is just a 'compatibility' layer for ComputeCPP until they provide full support for SYCL2020 specifications
+*/
+
+namespace cl::sycl {
+using access_mode = cl::sycl::access::mode;
+enum class target {
+	device = static_cast<int>(cl::sycl::access::target::global_buffer),
```

Comment:
I just checked and CCPP 2.5.0 actually uses `unsigned int` as the underlying type. But best use `std::underlying_type_t` here!